### DISCUSSION TITLE
Access Proxmox Virtual Environment with Teleport

### DISCUSSION THREAD

**Discussion by DaveTheTank** [gravitational/teleport#48630] [Q&A]
Hello Guys,

I am currently trying to make multiple Proxmox nodes accessible via Teleport. Right now, I am testing with one node. I followed the instructions to set up the app accordingly: https://goteleport.com/learn/how-to-access-proxmox-virtual-environment-with-teleport/

Questions:

1. Unfortunately, it states here that a wildcard certificate is required. I can't find any documentation on how to create a wildcard certificate using the standard Teleport ACME. Do I need to provide one myself?

3. I was able to successfully set up the app. However, when I try to access it now, I only get the message "This website is not reachable - Check if 'proxmox.teleport.mydomain.de' has a typo."

5. Can I only add one node? Once I change the name from "proxmox" in the teleport.yaml, my Teleport service doesn't start anymore.

Am I misunderstanding something here? Or can someone help me?

**Comment by webvictim** [comment]
1. You don't explicitly need a wildcard certificate - Teleport's built-in ACME support will automatically get you a cert for `<app-name>.teleport.example.com`, assuming that the app is publicly accessible and you have a matching DNS record pointed to the IP of your Teleport proxy.

2. This sounds like you don't have the DNS record for the app configured as mentioned above.

3. You can add as many nodes as you like, and you can add multiple apps served by one node if you want. See docs here: https://goteleport.com/docs/reference/agent-services/application-access/

```yaml
app_service:
  enabled: true
  apps:
  - name: proxmox
    uri: http://10.0.0.1
  - name: another-proxmox
    uri: http://10.0.0.2
```
